examples/getting-started-app/getting-started-app-scm-1.rockspec

```diff
@@ -7,9 +7,8 @@ source  = {
 dependencies = {
     'tarantool',
     'lua >= 5.1',
-    'luatest == 0.5.0-1',
     'ldecnumber == 1.1.3-1',
-    'cartridge == 1.2.0-1',
+    'cartridge == 2.0.1-1',
```

Comment:
`checks` is missed.

Maybe it would be better to update all this files by calling `cartridge create`?
Just to keep their actual.